src/pages/Homepage.tsx

```diff
@@ -1,3 +1,14 @@
-import { Example } from '../components/Example/Example';
-
-export const Homepage = () => <Example>Hello world!</Example>;
+import { Welcome } from './sections/Welcome/Welcome';
+import { HowDoesItWork } from './sections/HowDoesItWork/HowDoesItWork';
+import { CurrentlyReading } from './sections/CurrentlyReading/CurrentlyReading';
+import { UpcomingEvents } from './sections/UpcomingEvents/UpcomingEvents';
+import { Testimonials } from './sections/Testimonials/Testimonials';
```

Comment:
I recommend creating `components/homepage` folder (yes lower case `h`) which will have everything that is used ONLY in Homepage page.

Move the `sections outside of the `pages` folder. Let's use the `pages` only for . . . pages :D